resources/views/users/viewanswered.blade.php

```diff
@@ -8,35 +8,55 @@
                 <h2>Quiz Section</h2>
                 <p>Check Your Brain</p>
             </div>
-
-            @foreach ($quizAnswers as $answerKey => $quizAnswer)
-                @php
-                    $quizQuestion = \App\Models\QuizQuestion::where('id', '=', $quizAnswer->quiz_question_id)->first();
-                    $quizOptions = \App\Models\QuizOption::where('quiz_question_id', '=', $quizAnswer->quiz_question_id)->get();
-                @endphp
-
-                <h4> Q{{$loop->iteration}}.{{ $quizQuestion->question }}</h4>
-                <br>
-
-                @foreach ($quizOptions as $key => $quizOption)
-                    @php
-                        $quizpoint = \App\Models\QuizAnswer::where('quiz_option_id', '=', $quizOption->id)
-                            ->where('user_id', Auth::user()->id)
-                            ->first();
-                    @endphp
-
-                    @if ($quizpoint === null)
-                    <input type="radio" name="" value="">{{ $quizOption->option }}<br>
-                    @elseif($quizpoint->point === 1)
-                    <span class="bg-success text-white"><input type="radio" name=""
-                            value="">{{ $quizOption->option }}</span><br>
-                    @else
-                    <span class="bg-danger text-white"><input type="radio" name=""
-                            value="">{{ $quizOption->option }} </span><br>
-                    @endif
-                @endforeach
-            @endforeach
-
+            <div class="container mb-5">
+                <div class="row">
+                    <div class="col-12">
+                        @foreach ($quizAnswers as $answerKey => $quizAnswer)
+                            @php
+                                $quizQuestion = \App\Models\QuizQuestion::where('id', '=', $quizAnswer->quiz_question_id)->first();
+                                $quizOptions = \App\Models\QuizOption::where('quiz_question_id', '=', $quizAnswer->quiz_question_id)->get();
+                            @endphp
+                            <p class="fw-bold">Q{{ $loop->iteration }}.{{ $quizQuestion->question }}</p>
+                            <div>
+                                @foreach ($quizOptions as $key => $quizOption)
+                                    @php
+                                        $quizpoint = \App\Models\QuizAnswer::where('quiz_option_id', '=', $quizOption->id)
+                                            ->where('user_id', Auth::user()->id)
+                                            ->first();
+                                    @endphp
+                                    @if ($quizpoint === null)
+                                        <input type="radio" name="" value="">
+                                        <label for="one" class="box first">
+                                            <div class="course">
```

Comment:
Dynamic class name need.
class="box first bg-success text-white"
ex:
 bg-success instead of $bg_color;